sdks/typescript/src/v1/client/features/filters.ts

```diff
@@ -0,0 +1,33 @@
+import { HatchetClient } from '../client';
+/**
+ * The filters client is a client for interacting with Hatchet's filters API.
+ */
+export class FiltersClient {
+  tenantId: string;
+  api: HatchetClient['api'];
+
+  constructor(client: HatchetClient) {
+    this.tenantId = client.tenantId;
+    this.api = client.api;
+  }
+
+  async list(opts?: Parameters<typeof this.api.v1FilterList>[1]) {
```

Comment:
i think i do this in a number of places, but this is bad practice as most ides wont render the props

Reply:
yeahh just so easy to do this 😅 but I agree, I think we should file this away for later as something to improve